tools/docker/lib/common

```diff
@@ -100,7 +117,7 @@ CreateBuilder() {
     # just incase the env is not perfectly clean, remove any old instance of the builder
     docker builder rm certbot_builder || true
     # create the builder instance
-    docker buildx create --name certbot_builder --driver docker-container --driver-opt=network=host --bootstrap --use
+    docker buildx create --name certbot_builder --driver docker-container --driver-opt=network=host --bootstrap
     # add binfmt tools to the docker environment, with integration into the new builder instance
     InstallMultiarchSupport
```

Comment:
Now that we're calling this separately to be more explicit, I think we should delete these lines. Do you agree?
```suggestion
```

Reply:
you are correct